utils/extractLinks.js

```diff
@@ -0,0 +1,13 @@
+//Este archivo retorna links
```

Comment:
Si agregas comentarios que sea en ingles
```suggestion
// This file return links
```